gitalias.txt

```diff
@@ -187,6 +187,9 @@
   # log with one line per item.
   lo = log --oneline
 
+  # log with one line per item, reversed and pretty
+  ol = log --oneline --reverse --abbrev-commit --decorate --date=relative --format=format:'%C(bold cyan)%h%C(reset) - %C(bold green)(%ar)%C(reset) %C(white)%s%C(reset) %C(dim white)- %an%C(reset)%C(bold yellow)%d%C(reset)' -n 30
```

Comment:
We already have [`log --oneline --reverse`](https://github.com/GitAlias/gitalias/blob/db8e868077fdecd3be1e1e7091a8578e032a1aa9/gitalias.txt#L453) and in my opinion it's enough.

This alias has too many personal-style options. The colors suited for dark background but not light (at least not at my light grey terminal). Why `--date=relative`? Why `-n 30`? Why not 10 or 50?

My opinion is that such an alias is ok in your personal `~/.gitconfig` but not suitable for general public.

Reply:
Having a good one line log alias is a real help, especially those new to git.  The default behaviour hides many salient features of the current status, particularly branch names.  IMHO this is a useful behaviour.  However, feel free to reject the PR if you do not wish to adopt it.

Reply:
As I'm just an occasional contributor and reviewer I can neither accept nor reject the PR.